app/controllers/insights_cloud/hits_controller.rb

```diff
@@ -7,6 +7,7 @@ def index
 
       render json: {
         hits: hits,
+        isToken: Setting[:rh_cloud_token].length > 1,
```

Comment:
```suggestion
        hasToken: Setting[:rh_cloud_token].length > 1,
```
I think it describes the state, rather than what the response is.